gammapy/irf/effective_area.py

```diff
@@ -69,6 +69,25 @@ def plot(self, ax=None, energy=None, show_energy=None, **kwargs):
         return ax
 
     @classmethod
+    def from_parametrization(cls, energy, instrument='HESS'):
+        """Get parametrized effective area 
+
+        see :func:`~gammapy.irf.abramowski_effective_area`
```

Comment:
Update: I think examples on the docs pages of individual classes are really helpful, since we cannot know over which way users come to the page (they might click a link somewhere, and don't come via the ``gammapy.irf`` main page. so it's easier to have everything in one place.